src/language_names.cpp

```diff
@@ -71,7 +71,7 @@ size_t language_flags_from_name(const char *name)
       }
    }
 
-   return(0);
+   return(0); // will never been reached
```

Comment:
It could be reached if we call the function with an invalid/incorrect name. Not sure we should add this comment., although it should never be reached when things are right.